packages/react/src/components/organisms/Header/Header.stories.js

```diff
@@ -6,6 +6,7 @@ import MainNavData from 'MayflowerReactMolecules/MainNav/MainNav.knob.options';
 import UtilityNavData from 'MayflowerReactOrganisms/UtilityNav/UtilityNav.knob.options';
 import Header from './index';
 import HeaderDocs from './Header.md';
+import styles from './_index.scss';
```

Comment:
Once the global scss is broken out in scss
```
import componentStyles from './_index.scss';
import globalStyles from '../index.scss';
```
only show component css in storybook
```
<StoryPage styles={componentStyles} Description={HeaderDocs} />
```

Do we still need `styles/_index.scss`? If not, can we consolidate `components/index.scss` with `components/styles/index.scss`?